app/views/layouts/tylium.html.erb

```diff
@@ -62,7 +62,8 @@
     <%= render 'activities/poller' %>
 
     <!-- All modals go here -->
-    <%= render 'layouts/tylium/modals' %>
+    <%= render partial: 'layouts/tylium/image_modals' %>
+    <%= render partial: 'layouts/tylium/modals' %>
```

Comment:
if we already have a `modals` and you're adding a modal... let's render it from inside it...

... or you may have the wrong name? you're adding a single modal not a bunch of modals, so then perhaps what you need is a `tylium/modals/image` or `tylium/modals/image_preview` or something along those lines?